src/System.Runtime/tests/System/ExceptionTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using Xunit;
+
+public static class NullReferenceExceptionTests
+{
+    [Fact]
+    public static void NullReferenceException_Ctor1()
+    {
+        string defaultStr = "Object reference not set to an instance of an object.";
+        NullReferenceException i = new NullReferenceException();
+        Assert.Equal(defaultStr, i.Message);
+    }
+
+    [Fact]
+    public static void NullReferenceException_Ctor2()
+    {
+        NullReferenceException i = new NullReferenceException("Created NullReferenceException");
+
+        Assert.Equal("Created NullReferenceException", i.Message);
```

Comment:
Nit: I'd suggest pulling the string out into a local const, rather than writing it out twice.  Same goes for the subsequent tests.